TriangleApp/app/src/main/java/triangle/triangleapp/view/CameraPreview.java

```diff
@@ -0,0 +1,62 @@
+package triangle.triangleapp.view;
+
+import android.content.Context;
+import android.hardware.Camera;
+import android.util.Log;
+import android.view.SurfaceHolder;
+import android.view.SurfaceView;
+import java.io.IOException;
+
+/**
+ * Created by marco on 8-6-2017.
+ */
+
+public class CameraPreview extends SurfaceView implements SurfaceHolder.Callback {
+    private final String TAG = "CameraPreview";
+    private SurfaceHolder mHolder;
+    private Camera mCamera;
+
+    public CameraPreview(Context context, Camera camera) {
+        super(context);
+        mCamera = camera;
+
+        mHolder = getHolder();
+        mHolder.setType(SurfaceHolder.SURFACE_TYPE_PUSH_BUFFERS);
+        mHolder.addCallback(this);
+    }
+
+    public void surfaceCreated(SurfaceHolder holder) {
+        try {
+            mCamera.setDisplayOrientation(90);
```

Comment:
Created private int in class, init in constructor to prevent magic number.
private int mMediaRecorderMaxDuration;